codewof/programming/content/en/eight-is-great/question.md

```diff
@@ -0,0 +1,5 @@
+# Eight is great
+
+Create a function `eight_is_great(a, b)` that takes two integers, a and b, and returns `True` if either one is 8 or if their sum or difference is 8. Otherwise it should return `False`.
+
+For example `eight_is_great(15, 7)` returns `True` because the difference between 15 and 7 is 8.
```

Comment:
Again the new-sentence-on-new-line question and the should-return-be-bolded question